tests/test_loky_module.py

```diff
@@ -103,26 +118,28 @@ def test_only_physical_cores_error():
     if _cpu_count_user(cpu_count_mp) < cpu_count_mp:
         pytest.skip()
 
-    start_dir = os.path.abspath('.')
+    start_dir = os.path.abspath(".")
 
     with tempfile.TemporaryDirectory() as tmp_dir:
         # Write bad lscpu program
-        lscpu_path = tmp_dir + '/lscpu'
-        with open(lscpu_path, 'w') as f:
-            f.write("#!/bin/sh\n"
-                    "exit(1)")
+        lscpu_path = tmp_dir + "/lscpu"
+        with open(lscpu_path, "w") as f:
+            f.write("#!/bin/sh\n" "exit(1)")
```

Comment:
this one is not nice

```suggestion
            f.write("#!/bin/sh\nexit(1)")
```